CleverPush/Framework/CleverPush.framework/Versions/A/Headers/CleverPushInstance.h

```diff
@@ -50,6 +50,8 @@ typedef void (^CPFailureBlock)(NSError* error);
 
 typedef void (^CPAppBannerActionBlock)(CPAppBannerAction* action);
 
+typedef void (^CPAddRemoveTopicHandler)(NSString* message);
```

Comment:
Remove NSString* message parameter from callback, it should just be an empty void()